student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def intialize(classroom)
+    super()
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly add the parameters to the `initialize` constructor that you would be passed for the super class so it will be like:
```
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```
also, you need to change the parent(`Person`) initialize parameters to be 
`def intialize(age, name = 'Unknown', parent_permission = true)` 
instead of 
`def intialize(age, name = 'Unknown', parent_permission: true)`